eox_nelp/edxapp_wrapper/test_backends/site_configuration_m_v1.py

```diff
@@ -0,0 +1,10 @@
+"""Backend test abstraction."""
+from eox_nelp.edxapp_wrapper.test_backends import create_test_model
+
+
+def get_configuration_helpers():
+    """Return test class.
+    Returns:
+        Mock class.
+    """
+    return create_test_model('configuration_helpers', 'eox_nelp', __package__)
```

Comment:
configuration helpers is not a model,  so I'm not sure if this line will works in the future

Reply:
[7adf67d](https://github.com/eduNEXT/eox-nelp/pull/26/commits/7adf67d823de4083a57dc900b5855c179c91aaa2)